### PR TITLE
Follow-on cleanup to recent weak objects revert

### PR DESCRIPTION
**Pull request by boxtcarpel**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @boxtcarpel has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/Operations/BOXAPIAppUsersAuthOperation.m [hunk 1]
```diff
@@ -42,18 +42,19 @@ - (void)executeOperation
...
             [self.session.queueManager.delegate fetchAccessTokenWithCompletion:^(NSString *accessToken, NSDate *accessTokenExpiration, NSError *error) {
                 if (error == nil && accessToken && accessTokenExpiration) {
-                    weakSelf.accessToken = accessToken;
-                    weakSelf.accessTokenExpiration = accessTokenExpiration;
+                    self.accessToken = accessToken;
+                    self.accessTokenExpiration = accessTokenExpiration;
                 } else if (!error) {
-                    self.error = [[NSError alloc] initWithDomain:@"accessToken and accessTokenExpiration should be non-nil" code:BOXContentSDKAppUserErrorAccessTokenInvalid | BOXContentSDKAppUserErrorAccessTokenExpirationInvalid userInfo:nil];
+                    self.error = [[NSError alloc] initWithDomain:@"accessToken and accessTokenExpiration should be non-nil"
+                                                            code:BOXContentSDKAppUserErrorAccessTokenInvalid
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Operations/BOXAPIAppUsersAuthOperation.m:51]: Note, I changed the code here from a logical OR of two codes to a single error code, as it is meant to be used.

**Comment by boxtcarpel** [comment]
Test this please

**Comment by jpang1** [comment]
+2

**Pull request merged**